packages/rmf-dashboard-framework/docs/getting-started.md

```diff
@@ -0,0 +1,115 @@
+## Getting Started
+
+Welcome to RMF Dashboard Framework! This tutorial will guide you through the initial setup and provide a basic example to help you start building your dashboard application.
+
+### Setup
+
+Prerequisites:
+
+- pnpm
+- Latest Nodejs LTS
+
+Follow https://pnpm.io/installation to install pnpm, you can then install nodejs using pnpm
+
+```bash
+pnpm env use --global lts
+```
+
+Clone this repo and install deps
+
+<!-- TODO(koonpeng): install from npmjs after package is published. -->
+
+```bash
+pnpm create vite -t react-ts basic-dashboard
+cd basic-dashboard
+git clone --depth 1 https://github.com/open-rmf/rmf-web
```

Comment:
```suggestion
cd ~/
pnpm create vite -t react-ts basic-dashboard
cd ~/basic-dashboard
git clone --depth 1 https://github.com/open-rmf/rmf-web
```
I see that there are multiple `cd` commands in the instructions below, it will probably be best if we provide full paths to prevent any confusion

Reply:
I will make it more clear which directory the commands should be run from, but I don't think we should have the example pinned to the home directory.